### ISSUE TITLE
ENDF-reader example is broken

### ISSUE THREAD

**Issue by marc1uk** [pyne/pyne#1226]
Trying to run the [endf-reader example](https://pyne.io/examples/endf_reader.html) as-is on the website gives the following error when attempting to load the U235 data file:
`u235 = Library("U235-VII.txt")`
`xs_data = u235.get_xs(922350000, 16)[0]`
```
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-6-6fce4d341e13> in <module>()
----> 1 u235 = Library("U235-VII.txt")
      2 xs_data = u235.get_xs(922350000, 16)[0]
      3 
      4 #h1 = Library("H1-VII_mod.txt")
      5 ## args are nuclide ID, MT number, isotope ID (defaults to nuclide ID)

endf.pyx in pyne.endf.Library.__init__()

endf.pyx in pyne.endf.Library._read_headers()

ValueError: invalid literal for int() with base 10: '0  0'
```
After noticing that the printout from the evaluation info on the website was different to the files downloaded from the linked source, I tracked down a copy of the original files at [this location](http://ukrndc.kinr.kiev.ua/!!!site_ukrndc/DATA/ENDF-7_1/092_U_235). After comparing the two I found that the updated files seem be missing a trailing '0' in the first line of the file:
*File at the [example location](https://t2.lanl.gov/nis/data/data/ENDFB-VII-neutron/U/235)*
```
                                                                     1 0  0
 9.223500+4 2.330248+2          1          1          0          79228 1451    1
 0.000000+0 1.000000+0          0          0          0          69228 1451    2
 1.000000+0 2.000000+7          0          0         10          79228 1451    3
 0.000000+0 0.000000+0          0          0        733        1249228 1451    4
```
*Original file now at [this location](http://ukrndc.kinr.kiev.ua/!!!site_ukrndc/DATA/ENDF-7_1/092_U_235)*:
```
                                                                     1 0  0  0
 9.223500+4 2.330248+2          1          1          0          79228 1451    1
 0.000000+0 1.000000+0          0          0          0          69228 1451    2
 1.000000+0 2.000000+7          0          0         10          79228 1451    3
 0.000000+0 0.000000+0          0          0        733        1249228 1451    4
```
After adding the missing 0 to the end of the first line from the file at the original example linked location, the file loaded without a problem. 

I don't know whether this is a problem with the files in the web database or something the endf-reader should be robust against, but perhaps it would be sensible to link to a static, specific version of the data file that the reader supports. While this may not ensure the most up-to-date data (the location for more current sources could be suggested), it would ensure that the example will always work, and provides a reference file for comparison in the future.

**Comment by mehmeturkmen** [comment]
This issue is very peculiar to the raw format of ENDF/B-VII.0 library. Except for version 7.0, all other versions of ENDF, including version 8.0, have a "zero" number at the end of the first line of cross-section files. This issue can be solved quickly with a small modification in the file of `endf.pyx`.

**Comment by gonuke** [comment]
I don't think we need to change any code here!  It looks like there is a single case of this file missing an entry and possibly we should just not support that file since it is now outdated anyway?

**Comment by mehmeturkmen** [comment]
That is right. Version 7.0 will be obsolete soon. That was what I was trying to say yesterday. The code, and example in the tutorial, works correctly. Just maybe do we need to mention about this version specific problem some where in the tutorial? 
So I am closing this issue.

**Comment by gonuke** [comment]
We can also update the location from which we pull the ENDF file in the demo to get a file with the right format

**Comment by gonuke** [comment]
closed by #1297
